src/main/resources/userData/popularWorkouts.json

```diff
@@ -0,0 +1 @@
"description":null,"isSaved":true},{"exercises":[{"exercise":{"title":"Weighted Sit-ups","description":"Lie on a bench holding a weight plate across your chest. Sit up, engaging abs, then lower back down. Can also be done without weight","bodyParts":["Abdominal"],"equipment":["Bodyweight"],"difficulty":"Beginner","imagePath":"","timePerRep":3000,"type":null},"sets":3,"repsPerSet":5},{"exercise":{"title":"EZ Bar Preacher Curls","description":"Sit at a preacher curl bench with an EZ bar. Curl the bar up, keeping arms on the pad, then lower slowly.","bodyParts":["Biceps"],"equipment":["EZBar"],"difficulty":"Beginner","imagePath":"","timePerRep":3000,"type":null},"sets":3,"repsPerSet":5},{"exercise":{"title":"Straight Leg Calf Raises","description":"Sit on a leg press machine, place feet so the heels are outside the platform. Press with the top of your foot and let the weight come back slowly until you feel a stretch. Repeat","bodyParts":["Calves"],"equipment":["Machine"],"difficulty":"Beginner","imagePath":"","timePerRep":3000,"type":null},"sets":3,"repsPerSet":5},{"exercise":{"title":"Weighted Straight Pushups","description":"Get in a pushup position. Lower the chest to the ground and press up again. Can be done with equipped weights. Repeat with control.","bodyParts":["Chest","Triceps","Shoulders"],"equipment":["Bodyweight"],"difficulty":"Beginner","imagePath":"","timePerRep":3000,"type":null},"sets":3,"repsPerSet":5},{"exercise":{"title":"Pronated Cable Forearm Curls","description":"Face a low cable pulley with an overhand grip on the bar. Curl the weight up by curling the wrist and not bending the elbow, then lower slowly.","bodyParts":["Forearms"],"equipment":["Cable"],"difficulty":"Beginner","imagePath":"","timePerRep":3000,"type":null},"sets":3,"repsPerSet":5},{"exercise":{"title":"Machine Hip Thrusts","description":"Sit on a hip thrust machine, placing your upper back against the pad and your feet flat on the platform. Push through your heels to drive your hips upward, squeezing the glutes at the top, then lower back down.","bodyParts":["Glutes"],"equipment":["Machine"],"difficulty":"Beginner","imagePath":"","timePerRep":3000,"type":null},"sets":3,"repsPerSet":5},{"exercise":{"title":"Cable Stiff-Legged Deadlifts","description":"Stand facing a low pulley with a rope attachment between your legs. Keep knees soft and hinge at the hips to pull the rope up while maintaining a straight back. Return to standing.","bodyParts":["Hamstrings","Glutes","LowerBack"],"equipment":["Cable"],"difficulty":"Beginner","imagePath":"","timePerRep":3000,"type":null},"sets":3,"repsPerSet":5},{"exercise":{"title":"Machine Lower Back Curls","description":"Sit on a back extension machine, arms crossed. Bend at hips to curl forward, then extend back up.","bodyParts":["LowerBack"],"equipment":["Machine"],"difficulty":"Beginner","imagePath":"","timePerRep":3000,"type":null},"sets":2,"repsPerSet":5}],"name":"Full Body Workout (60 min)","description":null,"isSaved":true}]
```

Comment:
Typo in "Leg Worokout". Just CTRL+F you should be able to find it. Maybe even add 1 shorter fullbody workout (30 mins ish) and a longer one (90 mins)